src/fairseq2/models/wav2vec2/builder.py

```diff
@@ -418,6 +418,9 @@ class Wav2Vec2Config:
     """The maximum probability of masking a feature. Note that, due to mask span
     overlap, the effective probability might be smaller."""
 
+    min_num_spans: int
```

Comment:
For `Wav2Vec2Masker` the name `min_num_spans` is good enough since the context is clear, but in the general model configuration `min_num_spans` sounds a bit ambiguous. How about naming this `min_num_mask_spans` to be more descriptive?